### PR TITLE
✨  update text OSV

### PR DESCRIPTION
**Pull request by laurentsimon**
* **Please check if the PR fulfills these requirements**
- [] Tests for the changes have been added (for bug fixes / features)
- [X] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
cleanup

* **What is the current behavior?** (You can also link to an open issue here)
https://github.com/ossf/scorecard/pull/775

* **What is the new behavior (if this is a feature change)?**
https://github.com/ossf/scorecard/pull/775

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

* **Other information**:

### CODE REVIEW

File: checks/vulnerabilities.go [hunk 1]
```diff
@@ -105,7 +105,7 @@ func HasUnfixedVulnerabilities(c *checker.CheckRequest) checker.CheckResult {
 	vulnIDs := osvResp.getVulnerabilities()
 	if len(vulnIDs) > 0 {
 		c.Dlogger.Warn("HEAD is vulnerable to %s", strings.Join(vulnIDs, ", "))
-		return checker.CreateMinScoreResult(CheckVulnerabilities, "unfixed OSV detected")
+		return checker.CreateMinScoreResult(CheckVulnerabilities, "existing vulnerabilities detected")
```
Comment (inferno-chromium) [inline checks/vulnerabilities.go:108]: maybe add number of vulnerabilities and vuln ids in message, more information will be nice here?

X vulnerabilities detected: VULN_ID_1, VULN_ID_2

**Review by inferno-chromium — APPROVED** [review]
lgtm with a message change

File: checks/vulnerabilities.go (hunk 1 above)
Comment (laurentsimon) [reply checks/vulnerabilities.go:108]: the details are logged in the line above https://github.com/ossf/scorecard/blob/main/checks/vulnerabilities.go#L107.
The message is just the summary `reason`.

Comment (laurentsimon) [reply checks/vulnerabilities.go:108]: the details are logged in the line above https://github.com/ossf/scorecard/blob/main/checks/vulnerabilities.go#L107
The message `existing ...` is just the high-level summary we show in the `reason` column of the results.

**Pull request merged**